app/scripts/controllers/sign.ts

```diff
@@ -539,7 +548,7 @@ export default class SignController extends BaseControllerV2<
       AbstractMessageParamsMetamask
     >,
     updateState: (
-      state: SignControllerState,
+      state: Record<string, unknown>,
```

Comment:
Why can we not use the `SignControllerState`, that should be exactly what we're updating with these callbacks?

Reply:
I've added back the `SignControllerState`